common/v2/features/Dashboard/components/ActionTile.tsx

```diff
@@ -49,7 +49,7 @@ const SButton = styled(Button)`
     justify-content: space-between;
     text-align: left;
     & > img {
-      width: 54px;
+      height: 54px;
```

Comment:
can you comment what this change was for, just so that i know?

Reply:
Scaling of the icons was made by the width. Since trezor icon is not square and is taller than larger to keep uniformity I made them scale by their height. Doesn't change anything besides Trezor icon not being enormous